projects/packages/jetpack-mu-wpcom/src/features/wpcom-dashboard-widgets/wpcom-site-management-widget/index.js

```diff
@@ -41,34 +29,17 @@ const WpcomSiteManagementWidget = ( { siteName, siteDomain, siteIconUrl } ) => {
 				</div>
 				<div className="wpcom_site_management_widget__site-info">
 					<div className="wpcom_site_management_widget__site-name">{ siteName }</div>
-					<div className="wpcom_site_management_widget__site-url">{ siteDomain }</div>
+					<div className="wpcom_site_management_widget__site-url">
+						<a href={ siteUrl }>{ siteDomain }</a>
+					</div>
 				</div>
 				<div className="wpcom_site_management_widget__site-actions">
-					<a className="button-primary" href={ `https://wordpress.com/overview/${ siteDomain }` }>
-						{ __( 'Overview', 'jetpack-mu-wpcom' ) }
+					<a className="button-secondary" href={ `https://wordpress.com/overview/${ siteDomain }` }>
+						{ __( 'Hosting Overview', 'jetpack-mu-wpcom' ) }
+					</a>
+					<a className="button-secondary" href={ `site-editor.php` }>
```

Comment:
This assumes the site editor is always available, which is not the case depending on the theme.

Reply:
It's interesting that the button is always the same in My Home though:

https://github.com/Automattic/wp-calypso/blob/ce9621540e79b86a6a995a09b2f4eefaf47e6f32/client/my-sites/customer-home/cards/features/site-preview/index.tsx#L74-L78

But it points to `/site-editor/[domain]?canvas=edit`.

Reply:
I checked Atomic, and there's just no link for non block themes. So I added a condition.

<img width="380" alt="image" src="https://github.com/user-attachments/assets/41ba538c-954a-47c3-b389-a9ed04692ecb" />